setup.py

```diff
@@ -38,7 +38,7 @@
     py_modules=["bugbear"],
     zip_safe=False,
     python_requires=">=3.6",
-    install_requires=["flake8 >= 3.0.0", "attrs"],
+    install_requires=["flake8 >= 3.0.0", "attrs>=19.2.0"],
```

Comment:
As this is not a hard requirement for flake8-bugbear, I might move this to .travis.yml if anyone opens an Issue about needing to run it with an older attrs.